spec/lib/yelp/index.spec.js

```diff
@@ -0,0 +1,21 @@
+import { Location } from '../../../src/lib/yelp';
+
+const sfLocation = { city: 'San Francisco', stateCode: 'CA', countryCode: 'US' };
+const nyLocation = { city: 'New York City', stateCode: 'NY', countryCode: 'US' };
+const sfMexLocation = { city: 'San Francisco', stateCode: 'NAY', countryCode: 'MX' };
+const nyArgs = ['New York City', 'NY', 'US'];
+const sfMexArgs = [undefined, 'NAY', 'MX']; // San Francisco, Nayarit, Mexico.
+
+describe('Yelp', () => {
+  describe('Location', () => {
+    it('Defaults to San Francisco, CA, US', () => {
+      expect(new Location()).toMatchObject(sfLocation);
```

Comment:
`Location` class should be instantiated in the beginning of the describe block since its a dep of each of the following tests

Reply:
Each test uses a different Location instance

Reply:
Am blind 😂